src/utils.js/apiCalls.js

```diff
@@ -12,3 +12,21 @@ export const fetchAllReviews = () => {
       console.log(err);
     });
 };
+
+export const fetchCategoryNames = () => {
+  return ncGamesAPI
+    .get("/categories")
+    .then((response) => response.data.categories);
+};
+
+export const fetchReviewsByCategory = (categoryFilter) => {
```

Comment:
There is a more efficient way to query reviews using the axios params object in your fetchAllReviews function - definitely worth considering once you get to sorting reviews by multiple values